instagram/src/components/PostPage.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import styled from 'styled-components';
+import dummyData from '../dummy-data';
+import PostContainer from './PostContainer/PostContainer';
+import SearchForm from './SearchBar/SearchBar';
+import Fuse from 'fuse.js';
+
+const AppContainer = styled.div`
+  margin-top: 80px;
+`
+const LogOutButton = styled.button`
+  background-color: white;
+  font-size: 20px;
+  padding: 3px 10px;
+  position: fixed;
+  top: 55px;
+  right: 10px;
+  z-index: 2;
+  cursor: pointer;
+`
+
+class PostPage extends React.Component {
```

Comment:
Whatever you have could have been in App instead or App should have been a function component.